doc/source/notebooks/basics/classification.ipynb

```diff
@@ -494,7 +496,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can now plot the predicted decision boundary between the two classes. To do so, we can equivalently plot the contourlines $E[f(x)|Y]=0$, or $E[g(f(x))|Y]=.5$. We will do the later since it allows us to discover a the function `predict_y`:"
+    "We can now plot the predicted decision boundary between the two classes. To do so, we can equivalently plot the contour lines $E[f(x)|Y]=0$, or $E[g(f(x))|Y]=0.5$. We will do the latter since it allows us to introduce the function `predict_y`, which returns mean and variance at test points:"
```

Comment:
```suggestion
    "We can now plot the predicted decision boundary between the two classes. To do so, we can equivalently plot the contour lines $E[f(x)|Y]=0$, or $E[g(f(x))|Y]=0.5$. We will do the latter since it allows us to introduce the function `predict_y`, which returns the mean and variance at test points:"
```